kernelci/api/helper.py

```diff
@@ -385,26 +385,26 @@ def create_job_node(self, job_config, input_node,
                         runtime=None, platform=None):
         """Create a new job node based on input and configuration"""
         jobfilter = input_node.get('jobfilter')
+        platform_filter = input_node.get('platformfilter')
```

Comment:
```suggestion
        platform_filter = input_node.get('platform_filter')
```

Reply:
It is already correct, because inside node, when it is decoded from json to dict, it will be available by key platformfilter, but variable names according to PEP8 is platform_filter.

Reply:
But I see you have already changed the model field name in `kernelci.api.models` to `platform_filter`.
And the same will be used in the `input_node`.

Reply:
I used alias parameter, so while python variable is PEP8, field name in json is same as jobfilter.
https://docs.pydantic.dev/latest/concepts/fields/#field-aliases
I am not entirely sure, but it should work.

Reply:
Is this tested OK?
I don't think that field alias name would work as we use `allow_population_by_field_name` in model config option that would use field name instead of alias while getting or sending the object to DB.

Reply:
Yes, it seems it will use platform_filter due this parameter. I guess i will have to live with that and discrepancy in field names, we will migrate jobfilter later as well.